specification/security/get_role/types.ts

```diff
@@ -18,17 +18,25 @@
  */
 
 import {
-  ApplicationPrivileges,
-  IndicesPrivileges
+  ApplicationPrivileges, ClusterPrivilege,
+  IndicesPrivileges, RemoteClusterPrivileges, RemoteIndicesPrivileges
 } from '@security/_types/Privileges'
 import { RoleTemplate } from '@security/_types/RoleTemplate'
 import { Dictionary } from '@spec_utils/Dictionary'
 import { UserDefinedValue } from '@spec_utils/UserDefinedValue'
 import { Metadata } from '@_types/common'
 
 export class Role {
-  cluster: string[]
+  cluster: ClusterPrivilege[]
```

Comment:
same question (as above) here for the change from string[] to emum

Reply:
This type is as well only used in the response of `get_role`, so same situation as above.